cfme/common/vm.py

```diff
@@ -353,25 +373,8 @@ def open_console(self, console='VM Console', invokes_alert=False, cancel=False):
         # Click console button given by type
         view.toolbar.access.item_select(console, handle_alert=None
             if invokes_alert is False else True)
+        self.vm_console
```

Comment:
Q: can you please explain why vm_console is property ? and why you don't care of returned class ?

Reply:
the vm_console property of the vm class was designed by James Oden. He is currently on short term leave. but I remember he told me why he did that so and I don't remember that exactly. I think what he told me was as follows( if I recall correctly): that each VM will have only one console open at any given instance and hence it makes sense to make it as a property.